File: js/lv1/약수의 합.js
//[level 1] 약수의 합 120842
//https://school.programmers.co.kr/learn/courses/30/lessons/120842

//결과 
//정확성: 100.0
//합계: 100.0 / 100.0

function solution(n) {
    var answer = 0;
    
    for (let i = 1; i <= n; i++) {
        if (n % i === 0) {
            answer += i
        }
    }
    
    return answer;
}

File: py/Level1/약수의 합.py
#
# [level 1] 약수의 합
# https://school.programmers.co.kr/learn/courses/30/lessons/12928

# 결과
# 정확성: 100.0
# 합계: 100.0 / 100.0

def solution(n):
    sum = 0
    for i in range(1, n + 1):
        if n % i == 0:
            sum += i

    return sum